### PR TITLE
Update git-workflow.md

### PR DESCRIPTION
**Pull request by stevecurtis**
clearing up reviewer / assignation confusion. feel free to edit this text if it's not as clear as it should be.

### CODE REVIEW

File: docs/00-contributing/git-workflow.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ Resources:
 
 ## Assigning pull requests
 
-If you make a pull request related to a subject that is the responsibility of a specific team or person, please consider tagging or assigning it to that team or person (or pinging them in Slack to ask them to review it).  See also [docs-governance.md](docs-governance.md).
+If you make a pull request related to a subject that is the responsibility of a specific team or person, please consider assigning it to that team or person (or pinging them in Slack to ask them to review it). Review teams can be specified in the "Reviewers" section of the right hand sidebar, and pull requests can be assigned in the "Assignees" section. See also [docs-governance.md](docs-governance.md).
```
Comment (kevwalsh) [inline docs/00-contributing/git-workflow.md]: Thanks for helping to clarify this.  Review teams are actually automated, per docs-governance.md. I guess we should state that first, and not make it "see also".   Also, we're not using "assignees" yet in any formal way, i'm not exactly sure how that works, to be honest.  Reviewers have to approve it, but maybe if there's an assignee, only an assignee can actualy merge it? In any case, let's leave out the "assignee bit". 

So something like this: 

```Your PR will automatically be assigned to a team for a review, based on which part of the handbook you are editing. That automatic review assignment is explained in [docs-governance.md](docs-govenrance.md).  If you think that certain individuals on the team may have particular subject matter expertise or authority on the topic, you can also assign them directly via the "Reviewers" section of the right hand sidebar. ```

**Review by kevwalsh — CHANGES_REQUESTED** [review]

**Comment by kevwalsh** [comment]
@stevecurtis bumping this. i can implement this for you if you want.

**Comment by openprivacy** [comment]
@stevecurtis please implement (update this commit) with language equal or similar to what @kevwalsh wrote, and your PR will be accepted.

File: docs/00-contributing/git-workflow.md (hunk 1 above)
Comment (stevecurtis) [reply docs/00-contributing/git-workflow.md]: These changes are now being addressed in 
Update git-workflow.md #343. I'm resolving this PR.

**Pull request closed without merging**